src/request.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable import/no-cycle */
+
+import display from './bodyComponent';
+
+const requestURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/9GFpJH58W0GP2nThYOqo/scores';
+
+const getScores = async () => {
+  const response = await fetch(requestURL);
+  const list = await response.json();
+  return list;
+};
+
+const scoreAddition = (nameInput, scoreInput) => {
+  const postData = {
+    user: nameInput,
+    score: scoreInput,
+  };
+  fetch(requestURL, {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+    body: JSON.stringify(postData),
+  })
+    .then((response) => response.json())
+    .then((message) => {
+      if (message.result === 'Leaderboard score created correctly.') {
+        document.body.innerHTML = '';
+        display();
+      }
+    });
```

Comment:
**Changes Required**
Well done implementing the changes. However;
- [ ] Let us try to use `async `and `await` with the `fetch` method to send and receive data.

Reply:
This issue is resolved! ✔️